diff --git a/node_modules/sveltekit-zero-api/utils/string.js b/node_modules/sveltekit-zero-api/utils/string.js
index 41d8586..d3e61f4 100644
--- a/node_modules/sveltekit-zero-api/utils/string.js
+++ b/node_modules/sveltekit-zero-api/utils/string.js
@@ -1,4 +1,4 @@
-const invalidCharacters = ['[', '/', '\\', ']', '-', ':', '.', '+', '(', ')'];
+const invalidCharacters = ['[', '/', '\\', ']', '-', ':', '.', '+', '(', ')', ' '];
 /** Replaces `[` `/` `\` `]` `-` `:` `.` with `_` */
 export const toValidVariable = (name = '') => name.replaceAll(new RegExp(invalidCharacters.map(m => '\\' + m).join('|'), 'g'), '_');
 /** When importing files: We don't add `.ts` in the import-statement */
