diff --git a/node_modules/svelte-popover/src/Content.svelte b/node_modules/svelte-popover/src/Content.svelte
index 78622d7..0c9f6f4 100644
--- a/node_modules/svelte-popover/src/Content.svelte
+++ b/node_modules/svelte-popover/src/Content.svelte
@@ -11,16 +11,33 @@
   }
 </style>
 
-<div bind:this={contentRef} class="content" style="z-index: {zIndex + 10}; {positionStyle}">
-  <slot />
-  {#if arrow}
-    <div bind:this={arrowRef} class="arrow" style="position: absolute; color: {arrowColor}; {arrowStyleProps}">◥</div>
-  {/if}
-</div>
-
-<Overlay {zIndex} {overlayColor} {action} on:setOpen={setOpen} {stopPropagation} {preventDefault} />
-
+{#if body}
+  <div class="body-inject" use:inject={'body'}>
+    <div bind:this={contentRef} class="content" style="z-index: {zIndex + 10}; {positionStyle}">
+      <slot />
+      {#if arrow}
+        <div bind:this={arrowRef} class="arrow" style="position: absolute; color: {arrowColor}; {arrowStyleProps}">◥</div>
+      {/if}
+    </div>
+    
+    <Overlay {zIndex} {overlayColor} {action} on:setOpen={setOpen} {stopPropagation} {preventDefault} />
+  </div>
+  {:else}
+  <div bind:this={contentRef} class="content" style="z-index: {zIndex + 10}; {positionStyle}">
+    <slot />
+    {#if arrow}
+      <div bind:this={arrowRef} class="arrow" style="position: absolute; color: {arrowColor}; {arrowStyleProps}">◥</div>
+    {/if}
+  </div>
+  
+  <Overlay {zIndex} {overlayColor} {action} on:setOpen={setOpen} {stopPropagation} {preventDefault} />
+{/if}
 <script>
+  // @ts-nocheck
+	// https://www.npmjs.com/package/svelte-inject
+	import inject from 'svelte-inject'
+  export let body;
+
   export let arrowColor;
   export let targetRef;
   export let zIndex;
diff --git a/node_modules/svelte-popover/src/index.svelte b/node_modules/svelte-popover/src/index.svelte
index 6f15811..c3bbe0b 100644
--- a/node_modules/svelte-popover/src/index.svelte
+++ b/node_modules/svelte-popover/src/index.svelte
@@ -12,8 +12,14 @@
   <div bind:this={targetRef} class="target" style={open ? `z-index: ${zIndex + 10}` : ''} on:click={onClick} on:touchend={onTouchEnd} on:mouseover={onMouseOver} on:mouseout={onMouseOut}>
     <slot name="target" {open} />
   </div>
-  {#if open}
-    <Content on:open={onOpen} on:setOpen={setOpen} {placement} {targetRef} {zIndex} {arrow} {action} {overlayColor} {arrowColor} {preventDefault} {stopPropagation}>
+  {#if !persist}
+      {#if open}
+        <Content {body} on:open={onOpen} on:setOpen={setOpen} {placement} {targetRef} {zIndex} {arrow} {action} {overlayColor} {arrowColor} {preventDefault} {stopPropagation}>
+          <slot name="content" {open} />
+        </Content>
+      {/if}
+  {:else if targetRef}
+    <Content {body} on:open={onOpen} on:setOpen={setOpen} {placement} {targetRef} {zIndex} {arrow} {action} {overlayColor} {arrowColor} {preventDefault} {stopPropagation}>
       <slot name="content" {open} />
     </Content>
   {/if}
@@ -43,6 +49,8 @@
   export let stopPropagation = false;
 
   export let open = false;
+  export let persist = false;
+  export let body = false;
 
   const setOpen = () => {
     open = !open;
