diff --git a/SvelteTheme.svelte b/SvelteTheme.svelte
index d5602326ad8361f7568298019d131c638f705eaf..12576cf61e4b662f4e5732d2234ae9f2e5849f09 100644
--- a/SvelteTheme.svelte
+++ b/SvelteTheme.svelte
@@ -1,4 +1,6 @@
-<script >import { browser } from '$app/env';
+<script >
+//import { browser } from '$app/env';
+export let browser = undefined
 import { colorSchemes, MEDIA } from './constants';
 import { disableAnimation, getSystemTheme, getTheme } from './helpers';
 import themeStore, { setTheme } from './index';
diff --git a/SvelteTheme.svelte.d.ts b/SvelteTheme.svelte.d.ts
index 50e1a781ebd75961d5846e89dcb950bb850e24ed..bed3a383d428f529d2a857268f37cfe35b0bf407 100644
--- a/SvelteTheme.svelte.d.ts
+++ b/SvelteTheme.svelte.d.ts
@@ -12,6 +12,7 @@ declare const __propDef: {
         /** Mapping of theme name to HTML attribute value. Object where key is the theme name and value is the attribute value */ value?: {
             [themeName: string]: string;
         };
+        /** f*/ browser?: boolean | undefined;
     };
     events: {
         [evt: string]: CustomEvent<any>;
diff --git a/node_modules/.cache/@babel/register/.babel.7.5.5.development.json b/node_modules/.cache/@babel/register/.babel.7.5.5.development.json
new file mode 100644
index 0000000000000000000000000000000000000000..9e26dfeeb6e641a33dae4961196235bdb965b21b
--- /dev/null
+++ b/node_modules/.cache/@babel/register/.babel.7.5.5.development.json
@@ -0,0 +1 @@
+{}
\ No newline at end of file